### PR TITLE
HMA-7089: Added Pension Contribution

### PR DESCRIPTION
**Pull request by ngoulongkam**

### CODE REVIEW

**Review by chrisob55 — CHANGES_REQUESTED** [review]
It'd be great if we could also include the pension contributions in the [integration tests](https://github.com/hmrc/tax-kalculator/blob/main/src/jvmTest/kotlin/uk/gov/hmrc/calculator/ParameterizedCalculatorTests.kt). That way, we have very thorough tests.

**Comment by ngoulongkam** [comment]
@chrisob55 This PR is 1 of the 3 parts (Pension/Tapering/Student Loan).

Reason why I haven't added any integration test is because we still need the business to get back to us with some "validated" test data. 

This will probably come as part of the whole shebang with the last PR when all 3 parts are done.

**Comment by chrisob55** [comment]
> @chrisob55 This PR is 1 of the 3 parts (Pension/Tapering/Student Loan).
> 
> Reason why I haven't added any integration test is because we still need the business to get back to us with some "validated" test data.
> 
> This will probably come as part of the whole shebang with the last PR when all 3 parts are done.

Cool

File: src/commonMain/kotlin/uk/gov/hmrc/calculator/model/pension/AnnualPensionMethod.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.calculator.model.pension
+
+enum class AnnualPensionMethod {
+    PERCENTAGE, AMOUNT_IN_POUND
```
[the remark this answers is not in the document]
Comment (ngoulongkam) [reply src/commonMain/kotlin/uk/gov/hmrc/calculator/model/pension/AnnualPensionMethod.kt]: Address in [7644499](https://github.com/hmrc/tax-kalculator/pull/80/commits/7644499ccc5cc0f3eef75a60a294d30f99d5a51d)

Marking as resolved.

**Review by jemmaSlater — APPROVED** [review]

**Pull request merged**